core/embed/rust/src/ui/layout_eckhart/firmware/device_menu_screen.rs

```diff
@@ -129,6 +129,16 @@ enum Subscreen {
     AboutScreen,
 }
 
+// Used to preallocate memory for the largest enum variant
+#[allow(clippy::large_enum_variant)]
```

Comment:
Otherwise, build will fail due to https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant.